float64 time_s
float64 gx
float64 gy
float64 gz
float64 ax
float64 ay
float64 az
float64 odo_right_pos
float64 odo_left_pos
float64 odo_right_vel
float64 odo_left_vel
float64 opt_pos_x
float64 opt_pos_y
uint16 opt_quality
uint16 wall_distance_right
uint16 wall_distance_left
